fritzconnection/tests/test_soaper.py

```diff
@@ -123,6 +137,7 @@ def test_boolean_convert_fails(value):
 
 def test_long_error_message():
     response = Response()
+    response.status_code = 500
```

Comment:
Same here: a hard coded status code is not needed, because the content is xml.

Reply:
you're right ... this was a left-over from before